src/ngraph/ngraph_graph.h

```diff
@@ -163,7 +163,11 @@ inline std::string get_backend_name(const mxnet::Context &context) {
     return "GPU";
 #endif
   } else if (context.dev_type == mxnet::Context::CPU().dev_type) {
+#ifndef MXNET_USE_NGRAPH_IE
```

Comment:
I would switch this around to use `ifdef` rather than the negation.  Easier on the brain to think in the direct condition case.

Reply:
OK, will do.

Reply:
NOTE: @ransford2011 and @mbrookhart this is OK for now as "IE" will use the Inference Engine CPU backend which should always work.  However, we may need a more-better solution here as we turn on GPU, VPU, FPGA targets and want to run those with MXNet.  I don't know that I will have time to solve this problem.  Something for you guys to think about.